src/index.ts

```diff
@@ -1 +1,5 @@
+import { NodeProps, LinkProps } from './types'
+export type NodePropTypes = NodeProps
+export type LinkPropTypes = LinkProps
```

Comment:
After this morning, I've not find a better way 😢

Reply:
ok